### ISSUE TITLE
One context menu instead of two

### ISSUE THREAD

**Issue by grahamperrin** [Croydon/vertical-tabs-reloaded#187]
A frame from the screen recording at <https://github.com/Croydon/vertical-tabs-reloaded/issues/171#issuecomment-385476668>: 

![frame](https://user-images.githubusercontent.com/192271/39442660-ee88b746-4ca9-11e8-9e79-d7c107065bfd.png)

It was difficult to get that traditional context menu, I'm not sure how I got it. 

Is it possible for VTR context menu items to be appear as items in the traditional context menu?

**Comment by grahamperrin** [comment]
> … manage your tabs quickly …

Yep, but I don't want any single extension to preclude use of others. 

In reality (when not occasionally testing things with Firefox Quantum) I'm accustomed to this: 

![2018-05-01 19 13 50](https://user-images.githubusercontent.com/192271/39486407-e5a96786-4d73-11e8-94e5-a4a178d3ebbd.png)

The child approach seems normal to me. 

Maybe add a user preference?

**Comment by Croydon** [comment]
This context menu is the one from the a default Firefox tab right?

The tabs in VTR's sidebar are custom elements, it's impossible to get the default tab context menu there, instead you would get the default website element context menu (i.e. not tab related options, but website related options).

This isn't helpful, is it?

By there way, there are open bugzilla bugs to demand further APIs to handle these situations better, but this is the current state.

**Comment by grahamperrin** [comment]
Hi, 

> … from a default Firefox tab right? …

The shot was of a context menu for a tab of legacy Vertical Tabs Reloaded. So, "yes", I think :-)

> … This isn't helpful, is it? …

LOL, no worries. 

Does this add spice to the mix? 

[Custom Right-Click Menu – Add-ons for Firefox](https://addons.mozilla.org/addon/custom-right-click-menu/)

**Comment by Croydon** [comment]
Add-ons will be able to have a shared context-menu in the near future for custom elements, thanks to browser.menus.overrideContext()

This will improve the situation overall for the context menu(s)

**Comment by Croydon** [comment]
Re-opening as a bug, even though this is out of my hands for now.

Will be fixed via #205
